### PR TITLE
Add a '+' button to add the Pages 

### LINKED ISSUE #846: Manage Addition of pages using a '+' Button
We should be able to add pages to the project by clicking on a '+' button on the cards rather than directly clicking anywhere on the card. This would be more intuitive.

### PR DESCRIPTION
**Pull request by streetcoder123**
Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30515

**Changes**
- Add a '+' button to the pages card
- Remove the logic of adding pages by clicking anywhere on the card
- Add the logic of adding pages on clicking the '+' button

**Expected Look**
image

closes #846

### CODE REVIEW

**Comment by SahilTara** [comment]
Is there going to be a - button as well? Also can you add a picture?

**Comment by streetcoder123** [comment]
> Is there going to be a - button as well? Also can you add a picture?

Yes, there we shall be adding a '-' button to delete pages as well later. Picture Added :)

**Comment by annielovesjs** [comment]
Reminder to make the + bigger (remove the initial div and just style the button or just add '+' inside the div and have an onclick event instead of nesting a button)

File: src/client/src/containers/SelectableCard/index.tsx [hunk 1]
```diff
@@ -47,7 +49,10 @@ const SelectableCard = ({
   clickCount?: number;
   disabled: boolean | undefined;
   isFrameworkSelection: boolean;
+  isPagesSelection: boolean;
+  addPage: (idx: number) => void;
 }) => {
+  console.log(clickCount);
```
Comment (SahilTara) [inline src/client/src/containers/SelectableCard/index.tsx]: Is this needed?

**Comment by jcoc611-microsoft** [comment]
Waiting for some feedback from Christina (UX) on this one

**Review by SahilTara — COMMENTED** [review]
The number feels a bit weird there to be honest. Is there any final goal that you're working towards? Its a bit odd right now.

File: src/client/src/containers/SelectableCard/index.tsx (hunk 1 above)
Comment (streetcoder123) [reply src/client/src/containers/SelectableCard/index.tsx]: This was left unintentional. Shall remove it

**Comment by xtinah-w** [comment]
Hi all, we're working towards revising all the designs to look more like the mock I've created and have been separating UX tasks into small PRs  to make it more manageable for the rest of our team as they work parallel with other dev tasks. 

We're hoping to get the new style guide in with the new pages design and new card layout to address the drop off/observations we've made from user testing. 
v7-step3-pages

**Review by annielovesjs — APPROVED** [review]

**Review by jasmingoh — APPROVED** [review]

**Review by jsondoo — APPROVED** [review]
good work

**Pull request merged**